index.html

```diff
@@ -11,16 +11,59 @@
 <body>
 <div class="header section">
   <div class="container">
-    <h1>Final Project Template</h1>
-    <p>A starting point for final projects.</p>
+    <h1>Median Weekly Earnings Decline as Share of Women Increases</h1>
+    <p>Median Weekly 2015 Earnings of full time workers - Current Population Survey. From the <a href="http://www.bls.gov/cps/cpsaat39.htm"><font color="F3FF19">Bureau of Labor Statistics</font></a></p>
+  </div>
+</div>
+<!-- checkboxes http://getbootstrap.com/components/#btn-groups -->
+<div class="controls section">
+  <div class="container">
+    <div class="btn-toolbar">
+      <div class="btn-group">
+      <div class="filter">
+        <button type="button" class="btn btn-default filter" id="filter-agg">Aggregate Occupations</button>
+        <button type="button" class="btn btn-default filter" id="filter-detail">Detailed Occupations</button>
+    </div>
+    <div class="btn-toolbar">
+      <label for="soc-group">Detailed Occupational Group: </label>
+      <select id="soc-group">
+        <option value="all">All Occupational Groups</option>
+        <option value="mgt">Management</option>
+        <option value="busfin">Business and Financial Operations</option>
+        <option value="compmath">Computer and Mathematical</option>
+        <option value="eng">Architecture and Engineering</option>
+        <option value="sci">Life, Physical, and Social Science</option>
+        <option value="comm">Community and Social Service</option>
+        <option value="legal">Legal</option>
+        <option value="edu">Education, Training, and Library</option>
+        <option value="art">Arts, Design, Entertainment, Sports, and Media</option>
+        <option value="health">Healthcare Practitioners and Technical</option>
+        <option value="healthsupp">Healthcare Support</option>
+        <option value="protect">Protective Service</option>
+        <option value="food">Food Preparation and Serving Related</option>
+        <option value="maint">Building and Grounds Cleaning and Maintenance</option>
+        <option value="personal">Personal Care and Service</option>
+        <option value="sales">Sales and Related</option>
+        <option value="admin">Office and Administrative Support</option>
+        <option value="farm">Farming, Fishing, and Forestry</option>
+        <option value="const">Construction and Extraction</option>
+        <option value="instal">Installation, Maintenance, and Repair</option>
+        <option value="prod">Production</option>
+        <option value="trans">Transportation and Material Moving</option>        
+      </select>
+    </div>
   </div>
 </div>
 <div class="main section">
   <div class="container">
-    <div id="chart"></div>
+    <div class="row">
+      <div id="chart1" class="col-md-8"></div>
+      <div id="chart2" class="col-md-4 last"></div>
+      <img src="dollarbill.jpg" alt="A dollar bill" height=150 width=300/> 
```

Comment:
This image hasn't been added to git yet, so it appears broken for me. But thank you for the ALT text!